src/theme/styled.ts

```diff
@@ -7,3 +7,4 @@ import { Theme } from "./theme";
 
 export const styled = rawStyled;
 export const useTheme = () => useContext(ThemeContext) as Theme;
+export const SceneThemeUpdateContext = createContext<any>(undefined);
```

Comment:
Is this being used? I can't find where you are using this so if it isn't being used please remove it

Reply:
Please let me know about this.